src/rsz/src/Rebuffer.cc

```diff
@@ -700,12 +730,37 @@ int BufferMove::rebuffer(const Pin* drvr_pin)
           cannot do the propagation. However, if there is a
           junction which is just doing wiring (no buffers or loads) then fine.
          */
+
+        odb::dbModule* source_module = nullptr;
+        odb::dbITerm* drvr_iterm = nullptr;
+        odb::dbBTerm* drvr_bterm = nullptr;
+        odb::dbModITerm* drvr_moditerm = nullptr;
+        db_network_->staToDb(drvr_pin, drvr_iterm, drvr_bterm, drvr_moditerm);
+
+        if (drvr_iterm) {
+          dbInst* drvr_inst = drvr_iterm->getInst();
+          if (drvr_inst) {
```

Comment:
An iterm without an inst would be a serious db error.  I don't think you need to check for it here.